src/components/EditGrower.js

```diff
@@ -127,11 +131,20 @@ const EditGrower = (props) => {
             onSelectChange={handleChange}
           />
           {inputs.map((row, rowIdx) => (
-            <Grid item container direction="row" key={rowIdx}>
+            <Grid
+              item
+              container
+              direction="row"
+              justifyContent="space-between"
+              key={rowIdx}
+            >
               {row.map((input, colIdx) => (
                 <TextField
                   key={`TextField_${rowIdx}_${colIdx}`}
-                  className={classes.textInput}
+                  className={[
```

Comment:
This works fine, but the intention of defining `inputs` as a matrix of rows and columns is that it is easily extensible without styling each field explicitly. E.g., if `inputs` has three rows with 2, 1 and 3 fields, it would flex like this:

```
[   ] [   ]
[         ]
[ ] [ ] [ ]
```
I think a simple flexbox layout a small `gap` and simply `flex: 1` on the columns would work well.

Reply:
Thank you for your feedback. I understand your intention of defining inputs as a matrix of rows and columns to make it easily extensible without styling each field explicitly. I will make the necessary changes and update the pull request shortly.